vendor/github.com/samuel/go-zookeeper/zk/dnshostprovider.go

```diff
@@ -0,0 +1,88 @@
+package zk
+
+import (
+	"fmt"
+	"net"
+	"sync"
+)
+
+// DNSHostProvider is the default HostProvider. It currently matches
+// the Java StaticHostProvider, resolving hosts from DNS once during
+// the call to Init.  It could be easily extended to re-query DNS
+// periodically or if there is trouble connecting.
+type DNSHostProvider struct {
```

Comment:
That appears to be the case

On Thu, Sep 8, 2016 at 10:31 AM, Paweł Rozlach <notifications@github.com>
wrote:

> In vendor/github.com/samuel/go-zookeeper/zk/dnshostprovider.go
> <https://github.com/mesosphere/mesos-dns/pull/470#discussion_r78017476>:
>
> > @@ -0,0 +1,88 @@
> > +package zk
> > +
> > +import (
> > +	"fmt"
> > +	"net"
> > +	"sync"
> > +)
> > +
> > +// DNSHostProvider is the default HostProvider. It currently matches
> > +// the Java StaticHostProvider, resolving hosts from DNS once during
> > +// the call to Init.  It could be easily extended to re-query DNS
> > +// periodically or if there is trouble connecting.
> > +type DNSHostProvider struct {
>
> Please correct me if I got it wrong:
> This change will only affect people who use DNS to dynamically define ZK
> ensemble members, right ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/470/files/76cb2deb1618862fa8f3595590f6f22cffe958a1#r78017476>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLMXw8ueK28WZ94R_4WrYTKiPSED1ks5qoBxEgaJpZM4J2Q4l>
> .
>